src/main/java/org/example/App.java

```diff
@@ -36,6 +36,33 @@ public static void main( String[] args )
             connection = DriverManager.getConnection(url, username, password);
             UserService userService = new UserService(new UserDAO(connection));
             userService.getAllUsers();
+
+            User jonathan = new User("jt", "jt@gmail.com", "password", LocalDate.now(), null, null);
+//            userService.createUser(jonathan);
```

Comment:
I think it's better to remove commented lines in the final code

Reply:
Lines have been uncommented